waldronlab/bioconductor.yaml

```diff
@@ -15,8 +15,10 @@ manifest:
     docker_command: " "
     no_user: True
     no_network: True
+    dockerargs: "-v $HOME:/home/rstudio"
```

Comment:
FYI, you should get into the habit of using `docker_args` as I replaced `dockerargs` to be more consistent in style (both work for now)

Reply:
Ah, thanks @nsheff! Will do.